Client/utils/appAPI.js

```diff
@@ -0,0 +1,273 @@
+      AppActions.receiveSuccess(response.data.message);
+      AppActions.receiveErrors('');
+     // console.log(authuser);
+    })
+    .catch((error) => {
+      AppActions.receiveErrors(error.message);
+      // console.log(error);
+    });
+  },
+
+  confirmResetPassword(resetObject) {
+   // console.log(user);
+    axios.post('/users/confirmPasswordReset', {
+      code: resetObject.code,
+      newPassword: resetObject.newPassword
+    }).then((response) => {
+      AppActions.receiveSuccess(response.data.message);
+      AppActions.receiveErrors('');
+     // console.log(authuser);
+    })
+    .catch((error) => {
+      AppActions.receiveErrors(error.message);
+      // console.log(error);
+    });
+  },
+
+  signinGoogleUser(idToken) {
+   // console.log(user);
+    axios.post('/users/googleSignin', {
+      idToken
+    }).then((response) => {
+      // console.log(response);
+      const authuser = {
+        id: response.data.user.uid,
+        email: response.data.user.email,
+        profilePic: response.data.user.photoURL,
+        displayName: response.data.user.displayName,
+        isAuthenticated: true
+      };
+      AppActions.receiveSuccess(response.data.message);
+      AppActions.receiveAuthenticatedUser(authuser);
+      AppActions.receiveErrors('');
+    })
+  .catch((error) => {
+    AppActions.receiveErrors(error.message);
+  });
+  },
+
+
+  signoutUser() {
+    axios.post('/users/signout').then((response) => {
+      AppActions.receiveSuccess(response.data.message);
+    })
+    .catch((error) => {
+      AppActions.receiveErrors(error.message);
+    });
+  },
+
+  createNewGroup(group) {
+    console.log(group);
+    axios.post('/group', {
+      groupName: group.groupname,
+      dateCreated: group.datecreated,
+      createdBy: group.createdBy,
+      createdByDisplayName: group.createdByDisplayName,
+      createdByProfilePic: group.createdByProfilePic,
+      createdByUserId: group.createdByUserId
+    }).then((response) => {
+      //console.log('create group success');
+      AppActions.receiveSuccess(response.data.message);
+    })
+    .catch((error) => {
+     // console.log('create group error');
+      AppActions.receiveErrors(error.message);
+    });
+  },
+
+  addUserToGroup(user) {
+    axios.post(`/group/${user.groupId}/user`, {
+      email: user.email,
+      userId: user.userId,
+      username: user.username,
+      groupName: user.groupName
+    }).then((response) => {
+      AppActions.receiveSuccess(response.data.message);
+      // console.log(response);
+    })
+  .catch((error) => {
+    AppActions.receiveErrors(error.message);
+    // console.log(error);
+  });
+    // console.log('Adding user to group....');
+  },
+
+  postMessage(message) {
+    // console.log(message);
+    axios.post(`/group/${message.groupId}/message`, {
+      groupId: message.groupId,
+      messageBody: message.messageBody,
+      priority: message.priority,
+      postedon: message.postedon,
```

Comment:
Great work on following good variable naming convention. You however seem to have skipped that for 'postedon'.  Proper variable naming helps with code readability.